services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusResponse.kt

```diff
@@ -29,7 +31,7 @@ sealed interface HakemusDataResponse {
     val areas: List<ApplicationArea>
     val customerWithContacts: CustomerWithContactsResponse?
 
-    fun customersByRole(): Map<ApplicationContactType, CustomerWithContactsResponse>
+    fun customersByRole(): List<Pair<ApplicationContactType, CustomerWithContactsResponse>>
```

Comment:
Why not a map?

Reply:
A map is faster to understand while reading the code. And here it makes perfect sense since there can be only one customer per role.

Reply:
Changed it back to a map.